src/screens/TokensListScreen.tsx

```diff
@@ -80,8 +80,10 @@ export const TokensListScreen = ({ route, navigation }: Props) => {
 
   const filteredTokens = useMemo(() => {
     if (search.length > 1) {
-      return tokens.filter(({ description }) =>
-        description.toLowerCase().includes(search.toLowerCase())
+      return tokens.filter(
+        ({ token, description }) =>
```

Comment:
is there a chance the description does not have a value?

Reply:
I don't think so, this line on the create screen token blocks the creation if it's empty. https://github.com/nearform/optic-expo/blob/cdd26e79d0b59f6ea0f81e75f11f358159be24a0/src/screens/CreateTokenScreen.tsx#L46

It could be just whitespace but that wouldn't affect the functionality